semgrep-core/src/analyzing/Dataflow.ml

```diff
@@ -226,10 +227,24 @@ module Make (F : Flow) = struct
       fixpoint_worker eq mapping trans flow succs work''
 
   let forward_succs (f : F.flow) n =
-    (f#successors n)#fold (fun s (ni, _) -> NodeiSet.add ni s) NodeiSet.empty
+    (f.graph#successors n)#fold
+      (fun s (ni, _) -> NodeiSet.add ni s)
+      NodeiSet.empty
 
   let backward_succs (f : F.flow) n =
-    (f#predecessors n)#fold (fun s (ni, _) -> NodeiSet.add ni s) NodeiSet.empty
+    (f.graph#predecessors n)#fold
+      (fun s (ni, _) -> NodeiSet.add ni s)
+      NodeiSet.empty
+
+  let mk_worklist ({ graph; entry } : F.flow) =
```

Comment:
Could we add here a comment to explain that we are computing the set of reachable nodes in the CFG?